bootstrap/app.go

```diff
@@ -23,7 +23,38 @@ var waitForCompletion *sync.WaitGroup
 func getConfig(configPath string) (serviceconfig.ServiceConfig, error) {
 	cfg, err := NewConfigurationBuilder().
 		ConfigFilePath(configPath).
-		NoDefaults().
+		Defaults(serviceconfig.ServiceConfig{
```

Comment:
LGTM

On a slight tangent, as a user, I still don't know what I *need* to specify in my config file as required config. I wonder if there is a way of specifying what the minimum config is? The defaults kind of say, "we have these defaults, so you don't have to worry about specifying these," but we must have some other config that neither has a default nor is it strictly required. (Or we will have soon.)

Reply:
Added a comment block that shows what the minimal values are.